ota-plus-web/app/reactapp/src/stores/UserStore.js

```diff
@@ -89,7 +89,8 @@ export default class UserStore {
   deleteMemberFromOrganization = async (email, refetchMembers) => {
     resetAsync(this.deleteMemberFromOrganizationAsync, true);
     try {
-      const response = await axios.delete(`${API_USER_ORGANIZATION_DELETE_MEMBER}?email=${email}`);
+      const encodedEmail = encodeURIComponent(email)
```

Comment:
Add the ";" at the end please

Reply:
Ah too used to Scala 😄 Added now.
Just a question out of curiosity: semicolon here is optional right?

Reply:
Yup